src/apps/relay/mainrelay.c

```diff
@@ -188,7 +188,8 @@ turn_params_t turn_params = {
     {NULL, 0, {0, NULL}}, /*tls_alternate_servers_list*/
 
     /////////////// stop server ////////////////
-    0, /*stop_turn_server*/
+    DRAINMODE_NOT_ENALBED, /*drain_turn_server*/
```

Comment:
Enalbed -> enabled

Reply:
Thank you for reviewing so quickly!!! Nice find will fix it up.